vm/src/builtins/builtinfunc.rs

```diff
@@ -200,6 +202,13 @@ impl PyBuiltinMethod {
     fn doc(&self) -> Option<PyStrRef> {
         self.value.doc.clone()
     }
+    #[pymethod(magic)]
+    fn repr(&self, vm: &VirtualMachine) -> PyObjectRef {
```

Comment:
You can directly return a `String` here to simplify things.

Reply:
Done! Thanks!